.travis.yml

```diff
@@ -22,6 +22,15 @@ matrix:
       install: npm i
       script: npm run lint
 
+    - node_js: 10
+      env: TEST_SUITE=types
+      install: npm i
+      script:
+        - npm run build
+        - node scripts/release/cp-pkgs.js
+        - npx lerna link --force-local
```

Comment:
What I want to do is locally link all the newly prepared `@material/react-*` packages. This doesn't seem to do that; will poke more tomorrow.